.github/workflows/main.yaml

```
@@ -10,10 +10,8 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
-      # Cloud Functions supports only Go runtimes 1.11 and 1.13.
-      # Therefore, this test confirms that the source is able to build with Go 1.13.
       matrix:
-        go-version: ["1.13", "1.17"]
+        go-version: ["1.16", "1.17"]
```

Could you remain this comment?  
Without this comment, we don't know why testing with go 1.16.

```suggestion
      # Cloud Functions does not support the latest Go runtimes. Currently, it only supports up to Go 1.16.
      # Therefore, this test confirms that the source is able to build with the supported Go version.
      matrix:
        go-version: ["1.16", "1.17"]
```